tasks/linux.sh

```diff
@@ -51,17 +51,25 @@ case "$available_manager" in
     # `systemctl show` is the command to use in scripts.  Use it to get the pid, load, and active states
     # sample output: "MainPID=23377,LoadState=loaded,ActiveState=active"
     cmd_out="$("$s" "show" "$name" -p LoadState -p MainPID -p ActiveState --no-pager | paste -sd ',' -)"
-    success "{ \"status\": \"${cmd_out}\" }"
+
+    if [[ $action != "status" ]]; then
+      success "{ \"status\": \"${cmd_out}\" }"
+    else
+      enabled_out="$("$s" "is-enabled" "$name")"
+      success "{ \"status\": \"${cmd_out}\", \"enabled\": \"${enabled_out}\" }"
+    fi
     ;;
 
   # These commands seem to only differ slightly in their invocation
   "service"|"initctl")
     if [[ $s == "service" ]]; then
       cmd=("$s" "$name" "$action")
       cmd_status=("$s" "$name" "status")
+      enabled_out="$(chkconfig --list | grep "$name" | tr '\t' ' ')"
```

Comment:
@sensei-de-loafage Maybe you have a suggestion for cleaning up this output. To reproduce you can run it against a centos 6 node. Piping output of grep to tr replaces tabs characters which break json but there appear to still be a bunch of space. 
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ bolt task run service action=status name=rsyslog -n ed1og0hn8kr4h74.delivery.puppetlabs.net
Started on ed1og0hn8kr4h74.delivery.puppetlabs.net...
Finished on ed1og0hn8kr4h74.delivery.puppetlabs.net:
  {
    "status": "rsyslogd (pid  1672) is running...",
    "enabled": "rsyslog         0:off 1:off 2:on 3:on 4:on 5:on 6:off"
  }
Successful on 1 node: ed1og0hn8kr4h74.delivery.puppetlabs.net
Ran on 1 node in 0.99 seconds
```

Reply:
@donoghuc One suggestion would be to pass the name of the service directly to `chkconfig` instead of `grep`ping for it.

Regarding the output, looks like there isn't a way to explicit format it like `systemd` (one of the many problems it solved), but we could rely on word splitting to even out the spacing.  One of the only good things about word splitting is it will greedily eat up all whitespace between words.

```
[root@unm8ltz9vm1qs6y ~]# foo=($(chkconfig --list rsyslog))
[root@unm8ltz9vm1qs6y ~]# echo "${foo[@]}"
rsyslog 0:off 1:off 2:on 3:on 4:on 5:on 6:off
```

For some reason, after the first column there are spaces _and_ a tab, which is why it looks uneven.

Reply:
Just to note, that example relies on `IFS` being unchanged in the main script, which is it, so it should work.  If needed, `IFS` should always be set in a subshell like the example in the [using arrays](https://mywiki.wooledge.org/BashGuide/Arrays#Using_Arrays) section here does.